.devcontainer/devcontainer-lock.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
`.devcontainer/devcontainer-lock.json` was generated locally when starting the devcontainer.  
Should we commit this file to ensure reproducible environments, or keep it ignored?